hurumap_ke/templates/profile/profile_detail.html

```diff
@@ -1,841 +1,1018 @@
 {% extends 'profile/profile_detail.html' %} {% load humanize lookup staticfiles %}
 
 {% block profile_header_tail %}
-    <div class="column-full">
-        <p class="caption"><strong>Census data:</strong> 2009</p>
-    </div>
+  <div class="column-full">
+    <strong>{{ primary_releases.active.name }} {{ primary_releases.active.year }}</strong>
+    {% if primary_releases.other %}
+      <div class="tool-group toggle-sub-group release-list">
+        <a>Change release<i class="fa fa-chevron-circle-down"></i></a>
+        <ul class="sub-group" style="display: none;">
+          {% for release in primary_releases.other %}
+            <li>
+              <a href="{{ request.path }}?release={{ release.year }}">{{ release.name }} {{ release.year }}</a>
+            </li>
+          {% endfor %}
+        </ul>
+      </div>
+    {% endif %}
```

Comment:
The `Change release` link does appear but clicking it has no effect (doesn't display list of available releases).

![screenshot from 2019-01-16 10-30-12](https://user-images.githubusercontent.com/1779590/51233618-5a68cc00-197b-11e9-98c4-85f4d1cc2b5d.png)